packages/angular/projects/ui-angular/src/lib/primitives-poc/shared/constants.ts

```diff
@@ -0,0 +1,516 @@
+export const ComponentClassObject = {
+  Alert: {
+    className: 'amplify-alert',
+    components: ['Alert'],
```

Comment:
Looks like `components` is a React specific thing. Should we remove them here?

Reply:
Done.